src/GAP/oscar_to_gap.jl

```diff
@@ -21,3 +21,13 @@ GAP.julia_to_gap(obj::fmpz_mat) = GAP.julia_to_gap(Matrix(obj), recursive = true
 
 ## `fmpq_mat` to matrix of GAP rationals or integers
 GAP.julia_to_gap(obj::fmpq_mat) = GAP.julia_to_gap(Matrix(obj), recursive = true)
+
+## `GapGroup` to GAP group
+GAP.GapObj(obj::T) where T <: GAPGroup = return obj.X
+
+convert(::Type{GAP.GapObj}, obj::T) where T <: GAPGroup = GAP.GapObj(obj)
```

Comment:
FYI: This can (and should) be also expressed like this (same for the other places)
```suggestion
convert(::Type{GAP.GapObj}, obj:: GAPGroup) = GAP.GapObj(obj)
```

Reply:
Ok, I will change this. I am currently working on adding some tests.